web/src/components/Core/Page.js

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react'
+
+import CircularProgress from '@material-ui/core/CircularProgress'
+import Fade from '@material-ui/core/Fade'
+import { makeStyles } from '@material-ui/core/styles'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: theme.palette.background.default,
+    minHeight: '100vh',
```

Comment:
im pretty sure if we are using vh and vw we need to be sure to add this to the html

`<meta name="viewport" content="width=device-width, initial-scale=1">`

I can take that as an action item

Reply:
Nice, TIL